contents/docs/self-host/index.mdx

```diff
@@ -8,7 +8,7 @@ showTitle: true
 import Disclaimer from './\_snippets/disclaimer.mdx'
 import Sunset from '../self-host/\_snippets/sunset-disclaimer.mdx'
 
-> **This page covers our open-source Docker compose deployment**, which is available under a MIT license, [without guarantee](/docs/self-host/open-source/disclaimer). We are continuing to develop this version of PostHog, but please note that PostHog does not provide support for customers deploying this version beyond these docs.
+> **This covers our open-source Docker compose deployment**, which is available under a MIT license [without guarantee](/docs/self-host/open-source/disclaimer). We continue to develop this version, but do not provide support for it beyond these docs. We [no longer support K8s deployments](/blog/sunsetting-helm-support-posthog) and it is **no longer possible to buy licenses for self-hosted versions** - you must [migrate to PostHog Cloud](/docs/migrate/migrate-to-cloud). 
```

Comment:
```suggestion
> **This covers our free, open-source Docker compose deployment**, which is available under a MIT license [without guarantee](/docs/self-host/open-source/disclaimer). We continue to develop this version, but do not provide support for it beyond these docs. We [no longer support paid, open-source deployments](/blog/sunsetting-helm-support-posthog) - it is **no longer possible to buy licenses for self-hosted versions.** 
```

Reply:
Toned it down a wee bit - also 'you must migrate' was a bit much.

Reply:
```suggestion
> > **This covers our free, open-source Docker compose deployment**, which is available under a MIT license [without guarantee](/docs/self-host/open-source/disclaimer). We continue to develop this version, but do not provide support for it beyond these docs. We [no longer support paid, open-source deployments](/blog/sunsetting-helm-support-posthog) - it is **no longer possible to buy licenses for self-hosted versions** and we recommend users instead [migrate to PostHog Cloud](/docs/migrate/migrate-to-cloud). 
```

I want to get that migration link in there, as it's off in a separate section of the docs and will address the user need directly.